test_tools/sigma_verify_rules.py

```diff
@@ -76,8 +116,27 @@ def run_verifier(rules_path, config_file_path):
                     continue
 
                 rule_file_path = os.path.join(dirpath, rule_filename)
-                parsed_rule = sigma_util.get_sigma_rule(
-                    rule_file_path, sigma_config)
+
+                block_because_csv = False
+                for item in ignore['path']:
+                    if item in rule_file_path:
+                        return_rules_with_problems.append(rule_file_path)
+                        block_because_csv = True
+                        logging.info('{0:s} Ignoring because {1:s}'.format(
+                            rule_file_path, item))
+
+                if block_because_csv:
+                    continue
+
+                try:
+                    parsed_rule = sigma_util.get_sigma_rule(
+                        rule_file_path, sigma_config)
+                # This except is to keep the unknown exceptions
+                # this function is made to catch them and document
+                # them the broad exception is needed
```

Comment:
can you reword this comment?

something like
```
# Wide exception handling since there are multiple exceptions that can be raised by the underlying sigma library.
```

OR something like that, this documentation is just confusing...

Reply:
Oh yes, your suggestion is spot on